assets/js/pages/search.js

```diff
@@ -2,7 +2,7 @@
   window.reloadPackages = function(currentFunctionPage, currentPackagePage){
     $('html, body').animate({ scrollTop: 0 }, 'slow');
   	$.ajax({
-  		url: "/search_packages?q="+urlParam('q') + "&page=" + currentPackagePage,
+  		url: "/search_packages?q="+urlParam('q') + "&page=" + currentPackagePage +"&viewer_pane="+urlParam('viewer_pane'),
```

Comment:
I would try to keep RStudio related code in async_loader, why do we need `viewer_pane = 1` for thoses pages ?